astropy/utils/iers/iers.py

```diff
@@ -90,6 +90,10 @@ def download_file(*args, **kwargs):
     ``**kwargs`` after temporarily setting the download_file remote timeout to
     the local ``iers.conf.remote_timeout`` value.
     """
+    url = kwargs.get('remote_url', args[0] if len(args) > 0 else None)
```

Comment:
Where is this used?

Reply:
Oops!  Good catch, thanks @aarchiba  - that was left over from before the rebase. Will remove.